packages/rest/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@aries-framework/rest",
+  "version": "0.1.0",
```

Comment:
We should start with version 0.0.0. Then on the first release it will be bumped to 0.1.0